src/features/cards/cards.thunks.ts

```diff
@@ -260,22 +260,29 @@ export const getListOfDecklistsFromSearchTerm = createAsyncThunk(
 export const fetchDecklistById = createAsyncThunk(
   "decklist/fetchByIdStatus",
   async (
-    payload: { gameType: GameType; decklistId: number; position: Vector2d },
+    payload: {
+      gameType: GameType;
+      decklistId: number;
+      usePrivateApi: boolean;
+      position: Vector2d;
+    },
     thunkApi
   ) => {
     let response;
+    const privateApiUrl =
+      GamePropertiesMap[payload.gameType].privateDecklistApi;
+    const publicApiUrl = GamePropertiesMap[payload.gameType].decklistApi;
+    // if usePrivateApi is true and the game has a private decklist endpoint available, use it. Otherwise use public
+    const apiUrl =
+      payload.usePrivateApi && privateApiUrl ? privateApiUrl : publicApiUrl;
     try {
-      response = await axios.get(
-        `${GamePropertiesMap[payload.gameType].decklistApi}${
-          payload.decklistId
-        }`
-      );
+      response = await axios.get(`${apiUrl}${payload.decklistId}`);
     } catch (e) {
       thunkApi.dispatch(
         sendNotification({
           id: uuidv4(),
           level: "error",
-          message: `Couldn't load deck ${payload.decklistId}. Ensure the id is correct and not a private deck. Cardtable can only support public decks at this time.`,
+          message: `Couldn't load deck ${payload.decklistId}. If this is a private deck, ensure "Share your decks" is checked in the user's settings.`,
```

Comment:
Only change I'd suggest is here, I'd love if we could craft a message based on whether the game module supports private decks at all. Because LOTR LCG, for example, has the `Share your decks` feature but won't support private decks. 

So some like the original message if the game type doesn't support private deck loading, and the new message if it does.

Reply:
Actually thought of another change, if we _are_ trying to load a private deck and it fails, we can ask something similar, like "Ensure the id is correct and this is not a public deck id. If this is a private deck, ensure..."

Reply:
So basically 3 different error messages: 

- One for if the module doesn't support private decks and the request fails
- One if the module _does_ support private decks, and the request fails, and the user tried to load with the checkbox checked
- One if the module _does_ support private decks, and the request fails, and the user tried to load with the checkbox NOT checked